### ISSUE TITLE
Cannot proceed past initial dialogue with Icarus after giving Canned Sardines

### ISSUE THREAD

**Issue by Leroy8888** [HelixNebulaStudio/RiseOfTheDead#280]
When the player is introduced to Icarus and the canned sardines have been turned in. The dialogue will get stuck when a dialogue option has been selected. Icarus will still show his response, but the dialogue won't progress further then what was selected.

I will also like to include that Icarus will not reset to his introduction state if a data reset is used. Which makes this fairly difficult to replicate.

**Comment by MXKhronos** [comment]
Also having trouble reproducing this, may need more information on this. `/resethomenpc Icarus` if you need to reset your progress on Icarus. Thanks!

**Comment by Leroy8888** [comment]
I'll just leave a video of the issue here as it may or may not have been fixed in dev branch. It could be that this issue occured due to server performance or other factors which yet have to be discovered. Both example videos took place in the same server and the player managed to update to the next dialogue line by repeatedly rejoining the server.

https://cdn.discordapp.com/attachments/1233377096844185680/1233377097981100092/Roblox_2024-04-26_21-08-01.mp4?ex=664348ef&is=6641f76f&hm=24457afd424b474f161ea104f496fa9e41f5a66275eb11d0ff992c2ee9778135&

https://cdn.discordapp.com/attachments/1233377096844185680/1233378432621084774/Roblox_2024-04-26_21-22-28.mp4?ex=66434a2d&is=6641f8ad&hm=d194c9eea99ff47bcc3284341a38883bb45f07191f1134047f36b946373b3e38&

I've also noticed in live branch that the timer does not update once the canned sardines have been given.
https://gyazo.com/093a1eb2b5afffa5355faadda7338ea7
